src/safemode/BootOption.ts

```diff
@@ -0,0 +1,9 @@
+// SPDX-FileCopyrightText: 2024 Gnuxie <Gnuxie@protonmail.com>
+//
+// SPDX-License-Identifier: AFL-3.0
+
+export enum SafeModeBootOption {
```

Comment:
Hm wouldnt it be smarter to use something like this instead?

```
export const Snack = {
    Apple: 0,
    Banana: 1,
    Orange: 2,
    Other: 3,
} as const;
```

See also https://stackoverflow.com/a/78060803

Reply:
No, because enums that are numbers are meaningless while debugging.

Reply:
It would be cool if we could use symbols though or if there was a workaround.

Reply:
OH right

Reply:
Hm actually not sure if it must be numbers with this. In the end it will be transpiled to an object with readonly marked elements. So I would guess that Symbols work here. Dont have a PC to try that with me though

Reply:
IDK what would we gain from that?

Reply:
Cleaner transpiled code. The stackoverflow.com has some examples on which enuk variant transpiles to which thing

Reply:
Ah ok, do you have particularly strong opinions about that? I don't really feel a need to change it right now, but i'll have it in my mind to think about.

Reply:
No strong opinions :)